harvard-university-for-the-creative-arts.csl

```diff
@@ -426,9 +415,10 @@
       </if>
       <else>
         <names variable="author">
-          <name form="short" font-style="normal" and="text" delimiter-precedes-last="never" initialize-with="."/>
+          <name form="short" and="text" delimiter-precedes-last="never" initialize-with="."/>
           <et-al font-style="italic"/>
           <substitute>
+            <names variable="editor"/>
```

Comment:
please check that this is right -- but it seems implausible to not list editors of edited books in in-text citations (see the Hancke/Beyond Varieties of Capitalism example)

Reply:
That is probably my mistake. Usually you'd reference a chapter, so if referencing a edited book as a whole and not a chapter then yes I would like the editor to appear. Can you offer any guidance on what I need to change?

Reply:
I did change that already, so nothing; I was just asking you to confirm.